alpha/apps/kaltura/modules/extwidget/actions/serveFlavorAction.class.php

```diff
@@ -148,6 +151,8 @@ public static function serveLiveMediaSet($durations, $sequences, $playlistStartT
 			$mediaSet['liveWindowDuration'] = $dvrWindow;
 		}
 
+		$mediaSet['timeOffset'] = $offset;
```

Comment:
Maybe better to use the const in kSimuliveUtils::SCHEDULE_TIME_OFFSET_URL_PARAM - or maybe we need to move the const to here